tests/test_api.py

```
@@ -5239,3 +5240,195 @@ def test_oidc_verifier_jwks_stats() -> None:
+        headers={"Authorization": f"Bearer {operator_token}"},
+    )
+    assert response.status_code == 403, response.text
+
+
+def test_security_configuration_describe_cross_tenant_idor_blocked(monkeypatch) -> None:
+    """Same IDOR guard for the describe-by-id variant."""
+    from tests.conftest import issue_test_token
+
+    monkeypatch.setattr(
+        "sparkpilot.aws_clients.EmrEksClient.describe_security_configuration",
+        lambda self, env, cid: (_ for _ in ()).throw(
```

Ruff (0.15.14)

[warning] Unused lambda argument: `self`

(ARG005)